src/typings/dtos/order-input.dto.ts

```diff
@@ -1,6 +1,6 @@
 export class OrderInputDto {
   publicKey: string;
-  cardTransactionMode: "auth" | "authAndCapture" | "credit";
+  cardTransactionMode: 'auth' | 'authAndCapture' | 'credit' | 'verifyCard';
```

Comment:
ℹ️  You may want to update the docs ([this](https://docs.xmoney.com/api/reference/order/create-an-order#order/create-an-order/t=request&path=cardtransactionmode) and [this](https://docs.xmoney.com/guides/checkout/hosted-checkout#checkout-parameters)) to include `verifyCard` value.